common/changes/@itwin/core-frontend/geo-maplayers_auth_refactor_2022-01-10-14-48.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Revisited the way map layers authentication is handled in the UI.",
```

Comment:
Can you reword this in a way that means something to consumers of the package? i.e., be more specific about what actually changed?

Reply:
You are right, updated it to use the PR description instead.  Does that make sense to you?